conf/default.yaml

```diff
@@ -5,7 +5,9 @@ s3:
   secretAccessKey: '{{ENV.AWS_SECRET_ACCESS_KEY}}'
   region: '{{ENV.AWS_DEFAULT_REGION}}'
   profile: '{{ENV.AWS_PROFILE}}'
-
+minimumImageSize:
+  width: 10
```

Comment:
These should probably be set with env vars otherwise they will be hard to override. Also - might want them bigger than 10 x 10, especially if 10 is in px.